marlowe/doc/marlowe-data.adoc

```diff
@@ -83,15 +83,14 @@ type Timeout = Slot
 ----
 
 
-Accounts have a number `NumAccount` and an owner, who is a `Party` to the contract.
+Accounts have an owner, who is a `Party` to the contract.
```

Comment:
Each `Party` in the contract owns one account with the same name as the `Party`: